.travis.yml

```diff
@@ -2,21 +2,20 @@ language: node_js
 node_js:
     - "0.12"
     - "0.10"
-#   - "0.8"   # no longer supported by iconv
-#   - "0.6"   # no longer supported by async
-    - "iojs-v1.8.1"
-    - "iojs"
-
+    - "stable"
+    - "iojs-v1"
+    - "iojs-v2"
+    - "iojs-v3"
```

Comment:
I doubt we'll ever care to deploy to iojs 1, 2, or 3, and instead target node.js 4.0 (the unforked branch). So not much point in adding them here.